Content/SCPCB/GFX/Forest/rock/_NOTE_ROCK2.txt

```diff
@@ -0,0 +1,2 @@
+rock2's texture was missing, i think it was originally rockmoss.
+Well, this will be its texture now.
```

Comment:
This doesn't belong in the repo, mention it in a comment/commit message

Reply:
deleted